airbyte-integrations/connectors/source-mailchimp/unit_tests/test_source.py

```diff
@@ -1,116 +0,0 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
-#
-
-import logging
-
-import pytest
-from source_mailchimp.source import MailChimpAuthenticator, SourceMailchimp
-
-logger = logging.getLogger("airbyte")
-
-
-def test_check_connection_ok(requests_mock, config, data_center):
-    responses = [
-        {"json": {"health_status": "Everything's Chimpy!"}},
-    ]
-    requests_mock.register_uri("GET", f"https://{data_center}.api.mailchimp.com/3.0/ping", responses)
-    ok, error_msg = SourceMailchimp().check_connection(logger, config=config)
-
-    assert ok
-    assert not error_msg
-
-
-@pytest.mark.parametrize(
-    "response, expected_message",
-    [
-        (
-            {
-                "json": {
-                    "title": "API Key Invalid",
-                    "details": "Your API key may be invalid, or you've attempted to access the wrong datacenter.",
-                }
-            },
-            "Encountered an error while connecting to Mailchimp. Type: API Key Invalid. Details: Your API key may be invalid, or you've attempted to access the wrong datacenter.",
-        ),
-        (
-            {"json": {"title": "Forbidden", "details": "You don't have permission to access this resource."}},
-            "Encountered an error while connecting to Mailchimp. Type: Forbidden. Details: You don't have permission to access this resource.",
-        ),
-        (
-            {"json": {}},
-            "Encountered an error while connecting to Mailchimp. Type: Unknown Error. Details: An unknown error occurred. Please verify your credentials and try again.",
-        ),
-    ],
-    ids=["API Key Invalid", "Forbidden", "Unknown Error"],
-)
-def test_check_connection_error(requests_mock, config, data_center, response, expected_message):
```

Comment:
have we considered the tradeoffs between deleting the tests and updating the fixture to run them in the new implementation?